rcl/include/rcl/logging.h

```diff
@@ -90,6 +121,31 @@ RCL_PUBLIC
 RCL_WARN_UNUSED
 bool rcl_logging_rosout_enabled();
 
+/// Default output handler used by rcl.
+/**
+ * This function can be wrapped in a language specific client library,
+ * adding the necessary mutual exclusion protection there, and then used
+ * by calling `rcl_logging_configure_with_output_handler` instead of
+ * `rcl_logging_configure`.
+ *
+ * <hr>
+ * Attribute          | Adherence
+ * ------------------ | -------------
+ * Allocates Memory   | No
+ * Thread-Safe        | Yes
+ * Uses Atomics       | No
+ * Lock-Free          | Yes
+ *
+ * \return `TRUE` if logging rosout is enabled, or
+ * \return `FALSE` if logging rosout is disabled.
```

Comment:
@ivanpauno leftovers?

Reply:
Yes, I copied the doc block of the function above and forgot to clean up that part.
Fixed in https://github.com/ros2/rcl/pull/660/commits/c219b821e23206109c505fd512950293bb1f6b74.